templates/web/base/waste/bulky/choose_date.html

```diff
@@ -18,12 +18,7 @@
 [% END %]
 
 <form class="waste" method="post">
-  [% # 'Show later / earlier dates' functionality has been implemented but at the time of writing, Peterborough have requested it not be made available %]
-  [% PROCESS form override_fields = [
-    'chosen_date', 'continue',
-    'process', 'token', 'saved_data', 'unique_id'
-    ]
-  %]
+  [% PROCESS form %]
```

Comment:
VINDICATED.

Reply:
Hah, yes, I was quite pleased all your hard work wasn't in vain after all!